package.json

```diff
@@ -59,7 +59,7 @@
     "mocha": "^6.2.0",
     "mocha-junit-reporter": "^1.23.1",
     "ms": "^2.1.2",
-    "parse-function": "5.2.11",
+    "parse-function": "^5.4.3",
```

Comment:
I think we should use `"parse-function": "5.4.3"` to avoid updating to latest version whenever it is.

Reply:
Agree! let me update the version.